helm/cert-operator-chart/templates/deployment.yaml

```diff
@@ -16,6 +16,8 @@ spec:
         app: cert-operator
       annotations:
         checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+      annotations:
```

Comment:
Also, there are two `annotations` now.

Reply:
thx for checking closely! saved my butt :)